core/src/main/java/bio/terra/pearl/core/dao/participant/ParticipantUserDao.java

```diff
@@ -20,6 +22,16 @@ public Optional<ParticipantUser> findOne(String username, EnvironmentName enviro
       return findByTwoProperties("username", username, "environment_name", environmentName);
    }
 
+   public Optional<ParticipantUser> findOneByShortcode(String shortcode) {
+      return findByProperty("shortcode", shortcode);
+   }
+
+   public List<ParticipantUser> findAllWithMissingShortcode() {
```

Comment:
Can you just update `BaseJdbiDao.findAllByProperty` so that it checks if `columnValue` is null, and if so, uses a different query string?  (and findByProperty too while you're at it) -- those methods not behaving as expected for `null` has been a gotcha in our codebase for awhile

Reply:
That's a great idea, done in https://github.com/broadinstitute/juniper/pull/1070/commits/0364abb5a314b84dc8e41c7b4c8d59524623fe8e